### PR TITLE
Update roadmap for 1.13, 1.14.

### LINKED ISSUE #13222: README.md link to roadmap-2022.md and update roadmap
G'day
Love this product.  Please update the main README.md  to link direct to the new roadmap-2022.md  and also update the roadmap.

Also: Q: When will Terminal be bundled into Windows by default?
Thanks!

### PR DESCRIPTION
**Pull request by zadjii-msft**
I threw that Gannt chart that I whipped up a few weeks ago in here, but if we feel that should be pulled because it's more a set of [guidelines than actual rules](https://c.tenor.com/aeV80XD4CSgAAAAC/guidlines-pirates-of-the-caribbean.gif), then we can pull it. 

* [x] Closes #13222

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,10 +111,10 @@ repository.
 
 ---
 
-## Windows Terminal 2.0 Roadmap
+## Windows Terminal Roadmap
 
-The plan for delivering Windows Terminal 2.0 [is described
-here](/doc/terminal-v2-roadmap.md) and will be updated as the project proceeds.
+The plan for the Windows Terminal [is described here](/doc/roadmap-2022.md) and
+will be updated as the project proceeds.
```
Comment (lhecker) [inline README.md:117]: the?

File: doc/roadmap-2022.md [hunk 2]
```diff
@@ -22,20 +22,62 @@ Below is the schedule for when milestones will be included in release builds of
...
+        Release 1.17            :milestone, after c3, 0
+        1.17 becomes Stable     :milestone, after d3, 0
+    section Terminal 1.18
+        Features                :d1, after c2, 4w
+        Bugfix                  :d2, after d1  , 2w
+        Lockdown&bake           :d3, after d2  , 2w
+        Release 1.18            :milestone, after d3, 0
+```
+
+Again, these dates do not represent confirmed release dates, they are merely planning guidelines.
```
Comment (lhecker) [inline doc/roadmap-2022.md:80]: I'd remove this line.

**Comment by KotlinIsland** [comment]
I received a certified 'bruh' moment when I tried to look at the gantt chart 😳😳😳
image

File: doc/roadmap-2022.md [hunk 3]
```diff
@@ -22,20 +22,60 @@ Below is the schedule for when milestones will be included in release builds of
...
+    axisFormat  %d %b
+    section Terminal 1.14
+        Lock down & bake        :done, 2022-05-06, 2w
+        Release 1.14            :milestone, 2022-05-24
+    section Terminal 1.15
+        Features                :a1, done, 2022-05-06, 4w
+        Bugfix                  :a2, active, after a1  , 1w
+        Lock down & bake        :after a2  , 1w
+        Release 1.15            :milestone, 2022-06-21, 0
+        1.15 becomes Stable     :milestone, after b3, 0
```
Comment (KotlinIsland) [inline doc/roadmap-2022.md:60]: Features                :done, a1 , 2022-05-06, 4w
        Bugfix                  :active, a2, after a1  , 1w
        Lock down & bake        :after a2  , 1w
        Release 1.15            :milestone, 2022-06-21, 0
        1.15 becomes Stable     :milestone, after b3, 0

File: doc/roadmap-2022.md [hunk 4]
```diff
@@ -22,20 +22,60 @@ Below is the schedule for when milestones will be included in release builds of
...
+gantt
+    title Proposed Terminal Releases 1.14-1.18
+    dateFormat  YYYY-MM-DD
+    axisFormat  %d %b
+    section Terminal 1.14
+        Lock down & bake        :done, 2022-05-06, 2w
+        Release 1.14            :milestone, 2022-05-24
+    section Terminal 1.15
+        Features                :a1, done, 2022-05-06, 4w
+        Bugfix                  :a2, active, after a1  , 1w
```
Comment (KotlinIsland) [inline doc/roadmap-2022.md:57]:
```suggestion
        Features                :done, a1, 2022-05-06, 4w
        Bugfix                  :active, a2, after a1  , 1w
```

**Comment by codeofdusk** [comment]
> I threw that Gannt chart

It's spelled [Gantt](https://en.wikipedia.org/wiki/Gantt_chart).

**Pull request merged**